lib/magma/model.rb

```diff
@@ -1,30 +1,27 @@
-Sequel::Model.plugin :timestamps, update_on_create: true
-Sequel::Model.require_valid_table = false
-Sequel.extension :inflector
-
 class Magma
-  Model = Class.new(Sequel::Model)
-   
-  class Model
+  class Model < Sequel::Model
     class << self
       Magma::Attribute.descendants.
         reject { |attribute| attribute == Magma::ForeignKeyAttribute }.
         each do |attribute|
           define_method attribute.attribute_type do |attribute_name=nil, opts={}|
             @parent = attribute_name if attribute == Magma::ParentAttribute
-            attributes[attribute_name] = attribute.new(attribute_name, self, opts)
+            attributes[attribute_name] = attribute.new(opts.merge(
+              project_name: project_name,
```

Comment:
What is the behavior of 'new' and its relation to the (possibly non-existent) db entry here?

Reply:
From your comment below

> It seems like this situation persists with Magma::Attribute.new(), which creates an attribute object but does not create the corresponding record?

That's exactly what's happening. With `Magma::Attribute.new()`, we can load non-db backed attributes in memory. And then if we want to make changes, we can call `Magma::Attribute.update()` to insert that row into the attributes table.